### DISCUSSION TITLE
Migration From Cordova To CapacitorJs

### DISCUSSION THREAD

**Discussion by UnschooledGamer** [Acode-Foundation/Acode#1038] [General]
Logging It (I might add TODOs Later)

@bajrangCoder

**Comment by sebastianjnuwu** [comment]
I took a look at the community that uses it, it's not yet growing enough for the plugins and features that the current version has, so it wouldn't be very smart to change the entire system.

**Reply by bajrangCoder** [reply]
Hmm, right but Cordova sucks

**Reply by ByteJoseph** [reply]
> Hmm, right but Cordova sucks

Yes...

**Comment by UnschooledGamer** [comment]
An Attempt is made for migration at https://github.com/Acode-Foundation/Acode/pull/1266
